### DISCUSSION TITLE
How to set sampleCell parameter in addIterativeLSI function when subclustering less than 10000 cells?

### DISCUSSION THREAD

**Discussion by Dragonlongzhilin** [GreenleafLab/ArchR#781] [Questions / Documentation]
I have extracted a specific cluster and found it less than 10000 cells, which is default value of sampleCell in addIterativeLSI function.
Should I adjust this parameter to be close to my actual cell number？

**Comment by rcorces** [comment]
Please note that we no longer use the Issues section for questions on usage or feature requests. These belong in Discussions. Issues should only be used for Bug Reports. I have migrated your post to discussions.

The only real reason to use `sampleCellsPre` is to reduce memory usage and run time. So I would not bother reducing it. In fact, if the number of cells you have is less than the value provided to `sampleCellsPre` then it defaults to `NULL`. So you can set it to `NULL` if you want.
https://github.com/GreenleafLab/ArchR/blob/968e4421ce7187a8ac7ea1cf6077412126876d5f/R/IterativeLSI.R#L166

**Reply by Dragonlongzhilin** [reply]
thanks!
